pyproject.toml

```diff
@@ -50,12 +53,9 @@ click = "^8.0"
 freezegun = "^1.2.2"
 pytest-recording = "^0.12.1"
 
-[tool.poetry.group.dev.dependencies]
-ruff = "^0.0.246"
-docker = "^6.0.1"
-
 [tool.black]
 skip-string-normalization = true
+line-length = 100
```

Comment:
worth considering ditching skip-string-normalization configuration

Reply:
yeah, i'm torn between "no better time than now" and "this makes the change even noisier".

(i still like single quotes though, so i'm a bad judge) :grin: